src/components/home/FindYourBallotContainer.jsx

```diff
@@ -3,15 +3,17 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 // eslint-disable-next-line import/named
-import { geocodeAddress, lookupPrecinct } from 'server/services/arc_gis'
+import { geocodeAddress, geocodeFromMagicKey, lookupPrecinct } from 'server/services/arc_gis'
 
 import FindYourBallot from './FindYourBallot'
 
 export default class FindYourBallotContainer extends React.Component {
   state = {
     address: '',
+    selectedSuggestion: null,
     addressLookupResult: {},
     fetching: false,
+    suggestions: [],
```

Comment:
Note: the `suggestions` state isn't currently used. But it would be used again if we moved all the state back up to here.